packages/grafana-data/src/themes/createColors.ts

```diff
@@ -229,8 +229,8 @@ class LightColors implements ThemeColorsBase<Partial<ThemeRichColor>> {
   };
 
   gradients = {
-    brandHorizontal: 'linear-gradient(90deg, #FF8833 0%, #F53E4C 100%);',
-    brandVertical: 'linear-gradient(0.01deg, #F53E4C -31.2%, #FF8833 113.07%);',
+    brandHorizontal: 'linear-gradient(90deg, #26A97C 0%, #009B65 100%);',
+    brandVertical: 'linear-gradient(0.01deg, #009B65 -31.2%, #26A97C 113.07%);',
```

Comment:
I realize that the Grafana folks seem to be perfectly happy hardcoding hex values, but if this were a PR to one of our internal apps I would request that we make some constants for `NIGreen` and whatever `#26a97c` is. Feel free to dismiss this, as I don't know how much work it is to create and reference such a constant in this context.

Reply:
+1 I've also been wondering what it would entail to have constants for these hard-coded hex values

Reply:
Done.